cmake/Utils.cmake

```diff
@@ -29,25 +28,37 @@ function(msvc_use_static_runtime)
               set(${variable} "${${variable}}"  PARENT_SCOPE)
           endif()
       endforeach()
+      set(variables
+          CMAKE_CUDA_FLAGS_DEBUG
+          CMAKE_CUDA_FLAGS_MINSIZEREL
+          CMAKE_CUDA_FLAGS_RELEASE
+          CMAKE_CUDA_FLAGS_RELWITHDEBINFO
+      )
+      foreach(variable ${variables})
+          if(${variable} MATCHES "-MD")
+              string(REGEX REPLACE "-MD" "-MT" ${variable} "${${variable}}")
```

Comment:
What is this section for? Doesn't the previous section already replace `/MD` with `/MT` for MSVC?

Reply:
@RAMitchell Does MSVC use both `/MD` and `-MD` flags?

Reply:
@RAMitchell My guess is nvcc uses Unix like `-` syntax on Windows instead of `/`?  So one section for CXX and another for CUDA.  Notice the names of processed variables. `CXX` vs `CUDA`.

Reply:
Got it.